src/OgRoleInterface.php

```diff
@@ -5,11 +5,12 @@
  * Contains Drupal\og\OgRoleInterface.
  */
 namespace Drupal\og;
+use Drupal\user\RoleInterface;
 
 /**
  * Provides an interface defining an OG user role entity.
  */
-interface OgRoleInterface {
+interface OgRoleInterface extends RoleInterface {
```

Comment:
Yeah in my opinion it is a bug in PHP5, I mean the constants are defined in the interface, that is as abstract as it will ever get. But it seems like the PHP team sees it as a design choice.

But luckily it's fixed in PHP7!